packages/angular/src/generators/storybook-configuration/storybook-configuration.spec.ts

```diff
@@ -37,8 +37,8 @@ describe('StorybookConfiguration generator', () => {
     }));
     jest.spyOn(fileUtils, 'readPackageJson').mockReturnValue({
       devDependencies: {
-        '@storybook/addon-essentials': '^6.0.21',
-        '@storybook/react': '^6.0.21',
+        '@storybook/addon-essentials': '^6.2.9',
```

Comment:
Same here, we should use `~` instead of `^`. And yes, after you rebase with my upgrade PR we should use `~6.3.0` everywhere :)